packages/vue/test/base/tests/e2e/specs/breadcrumbs.cy.js

```diff
@@ -0,0 +1,9 @@
+describe("Components: Breadcrumbs", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:8080/components/breadcrumbs");
+  });
+
+  it("should have color attribute", () => {
+    cy.get('ion-breadcrumbs#color').should('have.prop', 'color');
```

Comment:
Don't need to address this now, but it looks like other frameworks would benefit from this check. Should this be a check in core instead of just in the Vue test app?

Reply:
I can move it over to core after.

Reply:
Created FW-3889 to track and assigned to you @thetaPC 🚀